src/librustc/hir/lowering.rs

```diff
@@ -173,15 +173,16 @@ enum ImplTraitContext<'a> {
     /// Example: `fn foo(x: impl Debug)`, where `impl Debug` is conceptually
     /// equivalent to a fresh universal parameter like `fn foo<T: Debug>(x: T)`.
     ///
-    /// Newly generated parameters should be inserted into the given `Vec`
+    /// Newly generated parameters should be inserted into the given `Vec`.
     Universal(&'a mut Vec<hir::GenericParam>),
 
-    /// Treat `impl Trait` as shorthand for a new universal existential parameter.
+    /// Treat `impl Trait` as shorthand for a new existential existential parameter.
```

Comment:
"existential existential"? 😄

Reply:
Yeah, this ain't no ordinary existential, it's super-powered!